robinhood.gs

```diff
@@ -33,7 +33,8 @@ function getAccessToken_() {
     'scope': "internal",
     'client_id': "c82SH0WZOsabOXGP2sxqcj34FxkvfnWRZBKlBjFS",
     'username': robinhoodUsername,
-    'password': robinhoodPassword
+    'password': robinhoodPassword,
+    'device_id': "<grab via login to Web>"  /* https://stackoverflow.com/questions/55902967/is-there-any-way-to-get-a-bearer-token-now-since-robinhood-has-changed-the-api/56841942 */
```

Comment:
@ParitoshKelkar You're correct, the cookie is called `device_id` in the web inspector, and should be `device_token` here

Use device_token for this field guys!